### ISSUE TITLE
Let's make OSBES as welcoming as possible!

### ISSUE THREAD

**Issue by fredster33** [jebbyk/OSBES-minecraft-bedrock-edition-shader#125]
This issue is meant to be a reference point for myself and possibly others, to make this repo as welcoming as possible.

Possible functions of this issue:

- Coordinate a contributor code of conduct draft
- Add more documentation
- Potentially bring on board additional maintainers

**Comment by OpenSauce04** [comment]
Yeah documentation is a bit of a weak spot, so that would be nice

**Comment by OpenSauce04** [comment]
Idea: what if we made a separate markdown document for contributing to the repo rather than clogging the readme?

**Comment by fredster33** [comment]
CONTRIBUTING.md, probably that would be nice.

**Comment by OpenSauce04** [comment]
Or alternatively, we could use the repo wiki?

**Comment by fredster33** [comment]
The wiki is pretty horrid. I could ?potentially? make a website. Wikis are bad for SEO, GH pages is pretty decent. Definitely there should be contributor health files, though, eg. CONTRIBUTING.MD, PULL_REQUEST_TEMPLATE.MD, CODE_OF_CONDUCT.MD, SECURITY.MD, CHANGELOG.MD, etc. Working on some of this this week. Thoughts on a website?

**Comment by fredster33** [comment]
Also, is OSBES looking for maintainers? Looks like the project can be spruced up a bit

**Comment by OpenSauce04** [comment]
> The wiki is pretty horrid. I could ?potentially? make a website. Wikis are bad for SEO, GH pages is pretty decent. Definitely there should be contributor health files, though, eg. CONTRIBUTING.MD, PULL_REQUEST_TEMPLATE.MD, CODE_OF_CONDUCT.MD, SECURITY.MD, CHANGELOG.MD, etc. Working on some of this this week. Thoughts on a website?

A website has already been in the works for a while
We are waiting until presets are released to make any more moves on it though

**Comment by OpenSauce04** [comment]
> Also, is OSBES looking for maintainers? Looks like the project can be spruced up a bit

I mean I seem to be the only one doing that, so more help would be appreciated if you feel up to it

**Comment by jebbyk** [comment]
> Also, is OSBES looking for maintainers? Looks like the project can be spruced up a bit

agreed :-)

**Comment by fredster33** [comment]
Just noticed that there's a few dupe issues regarding 1.17 support, they could probably be closed

**Comment by jebbyk** [comment]
ye working on it
